ansible_base/resource_registry/tasks/sync.py

```diff
@@ -139,7 +139,9 @@ def fetch_manifest(
     return [ManifestItem(**row) for row in csv_reader]
 
 
-def get_ansible_id_or_pk(assignment: AssignmentBase) -> str:
+def get_ansible_id_or_pk(assignment) -> str:
```

Comment:
are we dropping type hinting on purpose?

Reply:
Yes. Mostly. Importing `AssignmentBase` is, in fact, the problem. That would trigger an error if the RBAC app is not enabled. We don't want that.

We could type hint `models.Model`. I do this in many places, but I'm unsure if people are annoyed by it.